percona-monitoring-and-management/install.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Add the Percona Helm repository
+helm repo add percona https://percona.github.io/percona-helm-charts/
+
+# Update your local Helm chart repository cache
+helm repo update
+
+# Install the PMM  Helm chart
+helm install pmm percona/pmm --set secret.pmm_password=$PMM_ADMIN_PASSWORD --set service.type=$SERVICE_TYPE
+
+# Wait for pod to be ready
+kubectl wait pods --for condition=Ready --timeout=5m pmm-0
+
+# Getting public address for the next step
+public_address=$(kubectl get svc monitoring-service -o jsonpath="{.status.loadBalancer.ingress[0].ip"})
```

Comment:
Can we put a check on this in a loop until it gets the IP , after loadbalancer creation, it take take a few seconds to assign the IP.

Reply:
@spron-in another minor suggestion